apps/spruce/src/pages/projectSettings/tabs/GithubPermissionGroupsTab/FieldTemplates.tsx

```diff
@@ -0,0 +1,107 @@
+import { useRef } from "react";
+import styled from "@emotion/styled";
+import Button from "@leafygreen-ui/button";
+import { useLeafyGreenTable, LGColumnDef } from "@leafygreen-ui/table";
+import { Body } from "@leafygreen-ui/typography";
+import { ArrayFieldTemplateProps, ObjectFieldTemplateProps } from "@rjsf/core";
+import { PlusButton } from "components/Buttons";
+import Icon from "components/Icon";
+import { getFields } from "components/SpruceForm/utils";
+import { BaseTable } from "components/Table/BaseTable";
+import { size, tableColumnOffset } from "constants/tokens";
+import { Unpacked } from "types/utils";
+
+export const PermissionObjectFieldTemplate: React.FC<
```

Comment:
Nice, would be cool to genericize this at some point so we can move away from having inputs with repetitive labels!